privacy-config/privacy-config-impl/src/test/java/com/duckduckgo/privacy/config/impl/RealPrivacyConfigPersisterTest.kt

```diff
@@ -128,8 +150,32 @@ class RealPrivacyConfigPersisterTest {
         }
 
     @Test
-    fun whenPersistPrivacyConfigAndVersionIsEqualsThanPreviousOneStoredThenStoreNewConfig() =
+    fun whenPersistPrivacyConfigAndVersionIsLowerThanPreviousOneAndDifferentPluginsThenStoreNewConfig() =
```

Comment:
nit: I think the test name is wrong because it says we are storing the new config but we are testing the opposite, suggest to change to `whenPersistPrivacyConfigAndVersionIsLowerThanPreviousOneAndDifferentPluginsThenDoNothing`